src/openvic-simulation/map/ProvinceInstance.hpp

```diff
@@ -7,6 +7,7 @@
 #include "openvic-simulation/economy/production/ResourceGatheringOperation.hpp"
 #include "openvic-simulation/military/UnitInstance.hpp"
 #include "openvic-simulation/military/UnitType.hpp"
+#include "openvic-simulation/modifier/ModifierEffectCache.hpp"
```

Comment:
IMO it's better to forward declare `struct ModifierEffectCache`, including stuff which isn't absolutely necessary just increases compilation times